bootstrap_cfn/fab_tasks.py

```diff
@@ -622,7 +621,8 @@ def get_cloudformation_tags():
     """
     return {
         "Env": env.environment,
-        "Application": env.application
+        "Application": env.application,
+        "Bootstrap-cfn Version": bootstrap_cfn.__version__
```

Comment:
you'll probably need an import bootstrap_cfn somewhere, or use pkg_resources.require("bootstrap-cfn")[0].version directly